src/services/UserInfoRequestProcessor.ts

```diff
@@ -80,9 +80,6 @@ export class UserInfoRequestProcessor {
 			return new Response(HttpCodesEnum.SERVER_ERROR, "Server Error");
 		}
 
-		// add sessionId to all subsequent log messages
-		this.logger.appendKeys({ sessionId: sub });
```

Comment:
out of interest what's the reason for not using `appendKeys` here?

Reply:
I removed it because it was meaning that we logged the sub/user ID everywhere, that is considered PII no?

Reply:
In this case is the `sub` received in the JWT token a *subjectId* or a *sessionId* - see the comment on line 64. If it is a sessionId it is okay (and helpful) to log it.

Reply:
When I decode the JWT I just see `sub`. It is used in the personal-identity table to identify a person record but now I'm not 100% sure that that makes it the user ID.
![image](https://github.com/alphagov/di-ipv-cri-cic-api/assets/40401118/b6730c9f-4579-41dd-8745-8a15c207cbf3)

Happy to add it back in if we think it's safe!

Reply:
As discussed have added back in and renamed `sub` to `sessionId`